### PR TITLE
@joeyAghion => Fully drop the rsvps table and model for deprecation #migration

### PR DESCRIPTION
**Pull request by mzikherman**
Migration is to back up the database: already done.

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -24,7 +24,7 @@
     t.datetime "created_at",     null: false
     t.datetime "updated_at",     null: false
     t.string   "slug"
-    t.datetime "closes_at"
+    t.datetime "closes_at",      null: false
```
Comment (joeyAghion) [inline db/schema.rb:27]: This seems like an unrelated and potentially problematic change... it could fail if there are event records with no `closes_at` date, right?

**Comment by joeyAghion** [comment]
Thanks for this clean-up!

File: db/schema.rb (hunk 1 above)
Comment (mzikherman) [reply db/schema.rb:27]: I'm not really sure why `rake db:migrate` locally generated that. I think there should be `closes_at`, but I'll doublecheck.

Comment (mzikherman) [reply db/schema.rb:27]: This should be fine. There are no events with that field blank, and [the migration](https://github.com/artsy/rsvp-rails/blob/9f7b5efabb3d3165db74f068b2c0e1d6ae0f80e0/db/migrate/20161101185508_add_closes_at_to_events.rb#L3) that introduced it specified `null: false`. So I think this is correct.

**Comment by joeyAghion** [comment]
👍
